Description: configure: barometer: fix i2c lib usage for 4.0
Forwarded: https://github.com/collectd/collectd/issues/2646
Author: Andreas Hasenack <andreas.hasenack@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1742691
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=886920
Last-Update: 2018-02-21

diff --git Makefile.am Makefile.am
index ae027a368..ef71be836 100644
--- Makefile.am
+++ Makefile.am
@@ -591,7 +591,7 @@ if BUILD_PLUGIN_BAROMETER
 pkglib_LTLIBRARIES += barometer.la
 barometer_la_SOURCES = src/barometer.c
 barometer_la_LDFLAGS = $(PLUGIN_LDFLAGS)
-barometer_la_LIBADD = -lm
+barometer_la_LIBADD = -lm -li2c
 endif
 
 if BUILD_PLUGIN_BATTERY
diff --git configure.ac configure.ac
index e869a6a09..d9854d024 100644
--- configure.ac
+++ configure.ac
@@ -1883,7 +1883,7 @@ if test "x$ac_system" = "xLinux"; then
     [with_libi2c="no (symbol i2c_smbus_read_i2c_block_data not found - have you installed libi2c-dev ?)"],
     [[
       #include <stdlib.h>
-      #include <linux/i2c-dev.h>
+      #include <i2c/smbus.h>
     ]]
   )
 else
diff --git src/barometer.c src/barometer.c
index f698005ca..e3d418d2a 100644
--- src/barometer.c
+++ src/barometer.c
@@ -27,6 +27,7 @@
 
 #include <fcntl.h>
 #include <linux/i2c-dev.h>
+#include <i2c/smbus.h>
 #include <math.h>
 #include <stdint.h>
 #include <sys/ioctl.h>
