Author: Ruben Kerkhof <ruben@rubenkerkhof.com>
Subject: libcollectdclient: increase error buffer

diff --git a/src/libcollectdclient/client.c b/src/libcollectdclient/client.c
index 51a4ab238..3ae2e714b 100644
--- a/src/libcollectdclient/client.c
+++ b/src/libcollectdclient/client.c
@@ -99,7 +99,7 @@
  */
 struct lcc_connection_s {
   FILE *fh;
-  char errbuf[1024];
+  char errbuf[2048];
 };
 
 struct lcc_response_s {
