Description: mute liblvm2app depreciation warning
Forwarded: https://github.com/collectd/collectd/issues/2647
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=881641

Index: collectd-5.8.1/Makefile.am
===================================================================
--- collectd-5.8.1.orig/Makefile.am
+++ collectd-5.8.1/Makefile.am
@@ -1056,7 +1056,7 @@ endif
 if BUILD_PLUGIN_LVM
 pkglib_LTLIBRARIES += lvm.la
 lvm_la_SOURCES = src/lvm.c
-lvm_la_CPPFLAGS = $(AM_CPPFLAGS) $(BUILD_WITH_LIBLVM2APP_CPPFLAGS)
+lvm_la_CPPFLAGS = $(AM_CPPFLAGS) $(BUILD_WITH_LIBLVM2APP_CPPFLAGS) -Wno-cpp
 lvm_la_LDFLAGS = $(PLUGIN_LDFLAGS) $(BUILD_WITH_LIBLVM2APP_LDFLAGS)
 lvm_la_LIBADD = $(BUILD_WITH_LIBLVM2APP_LIBS)
 endif
